docs/Data_Portal/Users_Guide/cohort_builder.md

```diff
@@ -90,3 +90,40 @@ Users can then perform the following actions:
 
 * Download files associated with the cohort from the [Repository](Repository.md)
 * Analyze data from the cohort in the [Analysis Center](analysis_center.md)
+
+## Cohort Types
+
+Depending on how they are modified or created, cohorts can have different types of filters and thus behave differently:
```

Comment:
is the behavior difference mainly with regard to changes in data releases that may impact the cases returned or not returned? Perhaps a qualifier at the end should add: "... and thus behave differently with regards to which cases are included in the cohort"?

Reply:
yup! that's the main difference in behaviour! I'll edit.